bin/generate-manifest.js

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const packageMeta = require('../package.json');
+
+const manifest = {
```

Comment:
FWIW, I was also considering moving *everything* from manifest.json into package.json, since NPM seems pretty agnostic to custom properties in there. Then, this generation script could just scoop everything out of there and transform it as needed